### DISCUSSION TITLE
Cara mengganti komponen dalam kolom

### DISCUSSION THREAD

**Discussion by namiraprita** [teamalgoritma/community#247] [Q&A] [answered]
Dear Team Algoritma,

Ingin bertanya, bagaimana cara mengganti nama di kolom "experience levels" seperti ini:
- EN --> Entry Level
- MI --> Middle Level
- SE --> Senior Level
- EX --> Executive Level

![Screen Shot 2022-08-04 at 13 24 33](https://user-images.githubusercontent.com/107253301/182777741-d07ec287-704a-482e-984c-8b00a145d68b.png)

Terima kasih!

**Comment by kevbow** [comment, marked as answer]
Halo @namiraprita,

Anda dapat menggunakan fungsi `switch` yang digabungkan dengan fungsi `sapply()`. Contoh kode yang dapat saya berikan yaitu:
```
ds_us$experience_level <- sapply(X = as.character(ds_us$experience_level), 
FUN = switch,
"EN" = "Entry Level",
"MI" = "Middle Level",
"SE" = "Senior Level",
"EX" = "Executive Level")
```

Demikian yang dapat kami sampaikan, apabila jawaban ini sesuai dengan pertanyaan Anda harap menekan "Mark as answer". Terima kasih.
